src/Api/SecretsManager/Controllers/SecretsController.cs

```diff
@@ -246,4 +248,38 @@ public async Task<ListResponseModel<BulkDeleteResponseModel>> BulkDeleteAsync([F
         var responses = secrets.Select(s => new BaseSecretResponseModel(s));
         return new ListResponseModel<BaseSecretResponseModel>(responses);
     }
+
+    [HttpPost("organizations/{organizationId}/secrets/move")]
+    public async Task BulkMoveToProjectAsync(Guid organizationId, [FromBody] BulkMoveToProjectsRequestModel request)
```

Comment:
Can we write some unit tests for this controller logic/validation? 

Some examples in `test/Api.Test/SecretsManager/Controllers/SecretsControllerTests.cs`

Reply:
Added the one to check for SM access, let me know if you think it's missing any still.